doc/examples/petab.ipynb

```diff
@@ -0,0 +1,378 @@
q1iRu9ftVE7KW2+95Ri3AmGMaXJLt5Zw4yvLSI6P5fWfDye+ZZTbKdXLiO7tCQ+TZn+Z6YUXXnCMW4EwxjSp5dtLuOHlpSS1ieaNm04nMa6F2ynVW1x0JKd1bdvsB6rnzJnjGPdrgRCRMSKSJSLZInJvLfFficgGEVkjIp+ISGqNWJWIrPK+ZvszT2NM01iZu5frpi2jQ+to3rxpOB3iot1OqcHO6pnI2oL9lBxuvtNuxMbGOsb9ViBEJBx4HhgL9AWuEpG+x+22EshQ1VOBd4C/1IgdVdVB3tc4f+VpjGkaq/P28dOXltK+VRRv3jScjq2bb3GA6nEIVfgiu/n2IqZPn+4Y92cPYhiQrao5qloOzADG19xBVReo6hHvxyVAsh/zMca4ZG3+fq596SvatozkzZuGk9SmeRcHgFOT29I6unlPu/Hiiy86xv1ZILoAeTU+53u3ncjPgA9rfI4WkUwRWSIil9XWQERu9u6TWVTUfP9PMiZYVVZ5eOnzrVz1zyXERVcXh85tY9xOq1GEhwln9kzgs83Nd9qNefPmOcYDYsITEbkGyABG1dicqqoFIpIOfCoia1V1S812qjoVmAqQkZHRPP8fMiZILd1awoOz1rGp8CCjeiXy2IQBdAmS4nDMWT0TmbO2kC1Fh+nRoZXb6dRbZKTzg4n+LBAFQM0VsZO9275DREYDvwNGqWrZse2qWuD9mSMiC4HBwJbj2xtjAsvug6U8PmcTM1cW0KVtDFOuHcIFfTsiIm6n1uiGdWsHQOa2kmZZIF555RXHuD8LxDKgp4h0o7owTASurrmDiAwGpgBjVHV3je3xwBFVLRORBOAMvjuAbYwJMJVVHl5bsp2/fvw1ZZUeJv2gB7f9oAcxUYG16E9jSk9oSbuWUWRu38vEYV3dTqfeXCsQqlopIpOAj4BwYJqqrheRyUCmqs4GngBaAf/2/nWR671jqQ8wRUQ8VI+TPK6qG/yVqzGm4VSVj9bv4q/zsvh61yHO6pnAH8f1Iz2x+f1FXV8iwmld41mxvXk+Ub1w4ULHnp1fxyBUdQ4w57htD9Z4P/oE7RYDA/yZmzHm5Kgq/9u8h6c+zmJN/n7SE1rywk9OY0z/pKC8nHQiGWnxzN+4i+JDZbRv1fwe+nMSEIPUxpjm5aucYp76+GuWbiuhS9sY/nLFqUwY3IWI8NCbnCEjtXpp1OXb93JBvySXs6mff/7zn45xKxDGGJ9tKjzAo//dyGeb99AhrgUPj+/HlUO7EhUReoXhmP5d2hAVHtYsC0Rdk/VZgTDG+GT26h385p3VxESGc/9Fp3Dt8LSgHoD2VXRkOP27tCazGY5DzJ8/370xCGNM81flUZ78OIsXFm4hIzWeF64Z0iwn2POnjLR2vPLFNsoqq2gRETxFM3T7hcaYOh0oreDnry7jhYVbuPr0rrxx03ArDrUYkhpPeZWHdQX73U6lXv7+9787xq0HYYyp1ZaiQ9z0r0xyi4/wyGX9uWZ4at2NQtQQ70B15ra9DElt53I2vvvggw8c49aDMMZ8z4JNu7nsuS/Yf6SC139+uhWHOiS0akFa+9hmNw7x4YcfOsatB2GM+Y53l+dz9zur6de5NVOuzQi6+ZP8ZUhqOxZm7UZVg+Y5EOtBGGO+sXnXQX73/lqGd2vPv28ZacWhHjLS4ik+XM624iN17xwgnnnmGce4FQhjDAClFVVMemMlLaMieGbiILuFtZ4yvhmHKHE5E9998sknjnErEMYYAB757waydh3kqR8PpEMzX+3NDd0TW9EmJpLlzWgcYvZs59WcrUAYY5i7bifTl+Ry01ndOKd3B7fTaZbCwoTTurZtdgPVTqxAGBPiCvYd5TfvrOHU5Dbcc+EpbqfTrGWktSN79yH2HSl3OxWfPPnkk45xKxDGhLDKKg93vrmSKo/y7MTBIT2nUmM49jzEitzm0Yv48ssvHeP2r8GYEPbsJ5vJ3L6XxyYMIC2hpdvpNHsDk9sSESZkbmseBeLdd991jFuBMCZEfbmlmL8tyOaKIcmMH9TF7XSCQkxUOP26tAmacQgrEMaEoL2Hy7nrrZV0S2jJH8f1czudoDKkazyr8/ZRXulxO5U6Pf74445xKxDGhKC/L8ym6GAZz04cTMsWNqFCY8pIi6es0sP6HYE/cd+qVasc41YgjAkxRQfLeG3Jdi4b3IX+Xdq4nU7QqbnCXKCbMWOGY9wKhDEhZur/tlBe6eH2c3u6nUpQ6tA6mpR2Mc2iQNTFCoQxIaRm76Gb3bXkNxmp7cjcvhdVdTsVRw8//LBj3K8FQkTGiEiWiGSLyL21xH8lIhtEZI2IfCIiqTVi14nIZu/rOn/maUyosN5D0zgtNZ6ig2XklRx1OxVHWVlZjnG/FQgRCQeeB8YCfYGrRKTvcbutBDJU9VTgHeAv3rbtgIeA04FhwEMiEu+vXI0JBbsPllrvoYl8M3Hf9sCeuG/69OmOcX/2IIYB2aqao6rlwAxgfM0dVHWBqh6bG3cJkOx9fyEwT1VLVHUvMA8Y48dcjQl6UxflWO+hifTqGEdci4hm/zyEPwtEFyCvxud877YT+RlwbHkjn9qKyM0ikikimUVFRSeZrjHBa/fBUqZ/Zb2HphIeJgxOjWdFgBeIBx980DEeEIPUInINkAE8UZ92qjpVVTNUNSMxMdE/yRkTBKYuyqGiSrnDeg9NJiM1nqxdB9l/tMLtVE4oLy/PMe7PAlEApNT4nOzd9h0iMhr4HTBOVcvq09YYU7dveg+Duth8S01oSGo8qrAqb5/bqZzQyy+/7Bj3Z4FYBvQUkW4iEgVMBL6zOoWIDAamUF0cdtcIfQRcICLx3sHpC7zbjDH1NMXbe7j93B5upxJSBqW0JUxgeTNaYe54dRYIEQkTkR/X98CqWglMovoX+0bgbVVdLyKTRWScd7cngFbAv0VklYjM9rYtAR6musgsAyZ7txlj6mH3wVKmL7HegxtatoigT6fWLA/gqb/vu+8+x3idk7CoqkdE
```

Comment:
No refval for `k_exp_hetero` or need to extend plot limits?

Reply:
yeah it's on the axis exactly